Makefile

```diff
@@ -128,7 +128,7 @@ $(addprefix collect-templates-,$(SUPPORTED_VERSIONS)): collect-templates-%:
 	$(eval version:=$*)
 	mkdir -p ${TEMPLATES_OUT_DIR}/v${version}
 	cp ${RESOURCES_DIR}/smcp-templates/v${version}/${BUILD_TYPE} ${TEMPLATES_OUT_DIR}/v${version}/default
-	find ${RESOURCES_DIR}/smcp-templates/v${version}/ -maxdepth 1 -type f ! -name "maistra" ! -name "servicemesh" | xargs cp {} ${TEMPLATES_OUT_DIR}/v${version}
+	find ${RESOURCES_DIR}/smcp-templates/v${version}/ -maxdepth 1 -type f ! -name "maistra" ! -name "servicemesh" -exec rsync -av {} ${TEMPLATES_OUT_DIR}/v${version}/ \;
```

Comment:
Why do we need rsync instead of just cp? `-exec cp {} ...` seems to work on my Linux machine (whereas `xargs cp {} ...` doesn't; at least not in the 2.4 branch... but it does work in the 2.3 branch).

Reply:
Hey, if -exec cp {} works we can also use it, but rsync can be very convenient also, take into account that rsync is more efficient, and if the files exist there is going to only copy the modified part of the files. I know that it's not a complex copy operation but I do not see bad to use rsync

Reply:
The reason I'd like to avoid using rsync is becase I remember seeing an issue a few months ago because rsync wasn't available in one of the builder images. Also, I don't think rsync updates just a part of the file by default.

Reply:
OK, changed again and tested in my MacOs and working. Can you confirm that work also in your Linux @luksa ?